packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/Friend.cs

```diff
@@ -4,7 +4,7 @@
 
 using System;
 
-namespace UnbrandedTypeSpec.Models
+namespace UnbrandedTypeSpec
```

Comment:
I think this is related to another comment but use model namespace is on so we shouldn't be generating this into the root namespace

Reply:
I will take a look, we need to fix this.

Reply:
Fixed